src/LDNS.xs

```diff
@@ -18,8 +18,8 @@ to_idn(...)
 
             if (SvPOK(ST(i)))
             {
-               status = idna_to_ascii_8z(SvPVutf8_nolen(obj), &out, IDNA_ALLOW_UNASSIGNED);
-               if (status == IDNA_SUCCESS)
+               status = idn2_to_ascii_8z(SvPVutf8_nolen(obj), &out, IDN2_ALLOW_UNASSIGNED);
```

Comment:
The flags we se are important. Is IDN2_ALLOW_UNASSIGNED really what we want here? The features that we select are important. This has to be investigated first.

Reply:
The `IDNA_ALLOW_UNASSIGNED` has been used since [the beginning of the `to_idn()` function](https://github.com/zonemaster/zonemaster-ldns/commit/027f2ebfb382aec88742cef15c7bf9c0b00c1d82). I've just tried to keep the same flag thinking it would give the same behavior. Are you saying that there is a change in behavior?